pkg/rancher-desktop/utils/protocols.ts

```diff
@@ -77,8 +77,10 @@ export const protocolsRegistered = Latch();
  * production environments.
  */
 function setupAppProtocolHandler() {
+  // eslint-disable-next-line deprecation/deprecation
```

Comment:
Please add a comment on why this is needed. You already have the follow-up filed as #5659 (thanks!), so maybe mention that too.